en_ID/pages/assets/script.js

```diff
@@ -0,0 +1,29 @@
+const username = document.getElementById('input-username')
+const button = document.getElementById('btn-submit')
+const resultsContainer = document.getElementById('check-list')
+
+const checkPullRequests = async (value) => {
+  try {
+    const response = await fetch(
+      `https://api.github.com/search/issues?q=is:pr+repo:kulkultech/open-source+author:${value}`,
+    )
+    const results = await response.json()
+    return results
+  } catch (e) {
+    throw new Error(e)
+  }
+}
+
+const showData = () => {
+  checkPullRequests(username.value).then((res) => {
+    console.log(res)
+    const loop = res.items.map((user) => {
+      return `<li>${user.title}</li>`
+    })
+    resultsContainer.innerHTML = loop
```

Comment:
So the loop is an array of string, if you do the `resultsContainer.innerHTML = loop` directly the comma that coming from the array is inherited.